test/graphql-api/multipart.mocks.ts

```diff
@@ -0,0 +1,38 @@
+import { setupWorker, graphql } from 'msw'
+
+const worker = setupWorker(
+  graphql.mutation<
+    {
+      multipart: {
+        file1?: string
+        file2?: string
+        files?: string[]
+        plainText?: string
+      }
+    },
+    {
+      file1?: File
+      file2?: File
+      files?: File[]
+      plainText?: string
+    }
+  >('UploadFile', async (req, res, ctx) => {
+    const { file1, file2, files, plainText } = req.variables
+    const filesResponse: string[] = []
+    for (const file of files ?? []) {
```

Comment:
`??` allows `files` to equal a falsy value (i.e. `false`). This would still break the `for/of` loop:

```js
for (const file of false) // false is not iterable
```

I suggest we use `||` instead. Nullish coalescing operator (`??`) is useful when dealing with possibly falsy values, which isn't the case for `files`. If `files` is undefined or falsy we should always fallback to an empty array to not break the `for`/`of` loop.